charts/sealed-secrets-ui/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 name: sealed-secrets-ui
 description: A Helm chart for a Sealed Secrets web interface
 type: application
-version: 0.0.7
-appVersion: v0.1.3
+version: 0.0.8
+appVersion: v0.1.4
```

`version` and `appVersion` are auto-increments by workflow.

```suggestion
version: 0.0.7
appVersion: v0.1.3
```